packages/nouns-assets/src/utils.ts

```diff
@@ -32,3 +34,43 @@ export const getRandomNounSeed = (): NounSeed => {
     glasses: Math.floor(Math.random() * glasses.length),
   };
 };
+
+/**
+ * Emulate bitwise right shift and uint cast
+ * @param hex Hex representation of a number
+ * @param shiftAmount The amount to right shift
+ * @param uintSize The uint bit size to cast to
+ */
+export const hexShiftAndCast = (hex: string, shiftAmount: number, uintSize: number): string => {
+  hex = hex.replace(/0x/,'')
+  const start = hex.length - ((shiftAmount + uintSize)/4)
+  const end = hex.length - (shiftAmount/4)
+  return "0x" + hex.substring(start,end)
```

Comment:
Nit: Can use ether's BigNumbers for this as well:

```ts
const shifted = BigNumber.from(hex).shr(shiftAmount).toHexString();
return `0x${shifted.substring(shifted.length - (uintSize / 4))}`;
```

Reply:
Un-nitted!